server/routerlicious/packages/local-server/.eslintrc.cjs

```diff
@@ -8,11 +8,25 @@ module.exports = {
 		require.resolve("@fluidframework/eslint-config-fluid/minimal-deprecated"),
 		"prettier",
 	],
+
 	parserOptions: {
 		"project": ["./tsconfig.json", "./src/test/tsconfig.json"],
 		"promise/catch-or-return": ["error", { allowFinally: true }],
 
 		// TODO: enable strict null checks in tsconfig and remove this override
 		"@typescript-eslint/prefer-nullish-coalescing": "off",
 	},
+	rules: {
+		// This package uses node's events APIs.
+		// This should probably be reconsidered, but until then we will leave an exception for it here.
+		"import/no-nodejs-modules": ["error", { allow: ["events"] }],
+	},
+	overrides: [
+		{
+			files: ["src/localWebSocketServer.ts", "src/test/**/*.ts"],
+			rules: {
+				"import/no-nodejs-modules": "off",
+			},
+		},
+	],
```

Comment:
It doesn't actually use Node's `events`, it uses [the events package](https://www.npmjs.com/package/events) ([see package.json](https://github.com/microsoft/FluidFramework/blob/2498d517d0bceff9e66a64ba510a8745f68b0aad/server/routerlicious/packages/local-server/package.json#L69)) which looks the same to eslint, so it complains.

Let's do two things:
- Set the rule to error, remove the `allow` and the `overrides`, with a comment saying that this package runs in node but also in the browser so we don't want it to import Node packages.
- Import `events` with a different name like we do [here](https://github.com/microsoft/FluidFramework/blob/2498d517d0bceff9e66a64ba510a8745f68b0aad/packages/test/functional-tests/package.json#L88) (and update the imports in the ts files to look like [this](https://github.com/microsoft/FluidFramework/blob/2498d517d0bceff9e66a64ba510a8745f68b0aad/packages/test/functional-tests/src/test/containerRuntime.spec.ts#L8-L9)) so eslint doesn't trip up.

This applies to several packages updated in the PR which use the `events` package :).